eq-author-api/schema/tests/questionnaire.test.js

```diff
@@ -658,6 +702,19 @@ describe("questionnaire", () => {
       expect(deletedQuestionnaire).toBeNull();
       questionnaire = null;
     });
+
+    it("should not delete a locked questionnaire", async () => {
+      ctx = await buildContext({});
+      await setQuestionnaireLocked(
+        {
+          questionnaireId: ctx.questionnaire.id,
+          locked: true,
+        },
+        ctx
+      );
+
+      expect(deleteQuestionnaire(ctx, ctx.questionnaire.id)).rejects.toThrow();
+    });
```

Comment:
You could enhance this test, too, by asserting that the right error is thrown.

Reply:
Aye, cheers Matt - I did try and have them assert for a `ForbiddenError` but it doesn't actually percolate through to the tests - our (or Apollo's?) wrapper which executes the mutations for the tests always returns a plain old `Error` exception on failure rather the one I actually threw, so not much point checking for that.